BarrysPeanutsAkkaCascading/src/main/java/barryspeanuts/msg/Address.java

```diff
@@ -0,0 +1,73 @@
+package barryspeanuts.msg;
```

Comment:
```suggestion
package barryspeanuts.model;
```

This is a bit more common of a word for what this package is